src/check/parse/IR.zig

```diff
@@ -513,6 +514,11 @@ pub const Node = struct {
         /// Example: EXAMPLE
         /// * lhs - LHS DESCRIPTION
         /// * rhs - RHS DESCRIPTION
+        local_dispatch,
+        /// DESCRIPTION
+        /// Example: EXAMPLE
+        /// * lhs - node index of left expression
+        /// * rhs - node index of right expression
         bin_op,
         /// DESCRIPTION
         /// Example: EXAMPLE
```

Comment:
Are these placeholders? should we include the examples here as we go or is the plan to add these later sometime?